src/posix/platform/trel.cpp

```diff
@@ -561,10 +562,25 @@ void otPlatTrelResetCounters(otInstance *aInstance)
     ResetCounters();
 }
 
+void otSysTrelInit(const char *aInterfaceName) { platformTrelInitOnInterface(aInterfaceName); }
```

Comment:
What about rename `platformTrelInit` to `otSysTrelInit`?

Reply:
Done, moved the platformTrelInit into otSysTrelInit